packages/@o3r/stylelint-plugin/testing/mjs-resolver.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Jest resolver to replace .mjs imports with .mts for ESM compatibility
+ * Copied from https://github.com/kulshekhar/ts-jest/blob/main/e2e/native-esm-ts/mjs-resolver.ts
+ * @param path
+ * @param options
+ */
+const mjsResolver = (path, options) => {
+  const mjsExtRegex = /\.mjs$/i
+  const resolver = options.defaultResolver
+  if (mjsExtRegex.test(path)) {
+    try {
+      return resolver(path.replace(mjsExtRegex, '.mts'), options)
```

Comment:
```suggestion
      return resolver(path.replace(mjsExtRegex, '.mts'), options);
```
Missing linting on this file